plugin.rb

```diff
@@ -57,7 +57,8 @@ class Engine < ::Rails::Engine
   end
 
   add_to_serializer(:post, :reactions) do
-    reactions = object.reactions.map do |reaction|
+
+    reactions = object.reactions.reject { |hash| hash[:reaction_users_count] == nil }.map do |reaction|
```

Comment:
do we need to check on nil here ?

Reply:
Yaa, as we are taking like as a reaction, we take an entry in table with reaction_users_count = nil, so we avoid that reaction here

Reply:
No I mean ruby will do this without == nil no ?

Reply:
That's right 😅